src/components/Main/Results/OutcomeRatesTable.tsx

```diff
@@ -62,7 +64,7 @@ export function OutcomeRatesTable({ showHumanized, result, rates }: TableProps)
     Math.max(...result.deterministicTrajectory.map((x) => x.critical.total + x.overflow.total)),
   )
 
-  const totalFormatter = (value: number) => (showHumanized ? humanizeFormatter(value) : decimalFormatter(value))
+  const totalFormatter = (value: number) => formatNumber(value)
```

Comment:
This is causing some substantial inaccuracy of the tabular results.  e.g for US simulation, the total dead is 1.67M, that's being rounded all the way up to 2M.  Would it be better to round to a couple of decimal places?  330k difference is quite big.

Reply:
One option could be to change the rounding bool, to a number indicating number of decimal places, which would keep the same functionality possible, with more flexibility

Reply:
Ikes. Thats not good, I'll re-open the issue. Thanks for spotting it.